components/Books/CatchLog/CatchLog.js

```diff
@@ -61,7 +61,7 @@ export default function CatchLog() {
         >
           <Text style={styles.delBtn}>X</Text>
         </TouchableOpacity>
-        {/* {cloudinary_urls?.length && image} */}
+        {cloudinary_urls && image}
```

Comment:
cloudinary_urls, even as an empty array in the case of a missing photo, will always be true. The only thing stopping it from rendering anything is the fact that the image variable is being assigned null if the urls array is empty.